lblod/frontend-mow-registry
# GN-4463: add option to edit definition of road-markings

## Overview
This PR adds the option to edit the definition of a road-marking concept. This definition is (among other uses) used to filter road-markings on.

Additionally, this PR also adds changeset validations to the definition of road-markings and traffic-lights.

##### connected issues and PRs:
Solves [GN-4463](https://binnenland.atlassian.net/jira/software/c/projects/GN/boards/4?modal=detail&selectedIssue=GN-4463) partially

### How to test/reproduce
- Select an existing road-marking or create a new one
- Notice that it is possible to fill in a definition. The definition is a required field.
- The definition should also show up on the detail page of a road-marking
- When adding a related road-marking to a road-marking/traffic-light/road-sign, it should now correctly filter on the definition.

### Challenges/uncertainties
- This PR does not add the option to edit the definition of road-signs, the reason therefore is two-fold:
    * Road-sign-concepts do not include a definition in the mu-cl-resources data-model, so unsure if a definition is needed
    * The definition of road-sign-concepts is not used in the frontend (road-sign-concepts are filtered in a different manner)

### Checks PR readiness
